src/javaharness/java/arcs/android/demo/service/ArcsService.java

```diff
@@ -65,6 +70,20 @@ public void onCreate() {
     uiBroker.registerRenderer("notification", notificationRenderer);
   }
 
+  @Override
+  public void onDestroy() {
+    Log.d(TAG, "onDestroy()");
+    super.onDestroy();
+  }
+
+  @Override
+  public int onStartCommand(Intent intent, int flags, int startId) {
+    super.onStartCommand(intent, startId, startId);
```

Comment:
did you mean `super.onStartCommand(intent, flags, startId);`?

Reply:
Good catch!